### ISSUE TITLE
Plugin svelte: "showNotification" is declared in a module script and will not be reactive

### ISSUE THREAD

**Issue by Dexterp37** [mozilla-rally/rally-core-addon#463]
```
src/main.js → public/build/bundle.js...
(!) Plugin replace: @rollup/plugin-replace: 'preventAssignment' currently defaults to false. It is recommended to set this option to `true`, as the next major version will default this option to `true`.
(!) Plugin svelte: "showNotification" is declared in a module script and will not be reactive
src/routes/current-studies/StudyCard.svelte
88: 
89: // when the joined part changes, do something.
90: $: showNotification(joined, key);
       ^
91: $: isActive = $activeKey !== undefined && $activeKey === key;
92: 
created public/build/bundle.js in 5.2s
```

**Comment by Dexterp37** [comment]
@hamilton do you know what is this about?

**Comment by hamilton** [comment]
@Dexterp37 yes, this is because Svelte is listening to changes to the `showNotification`, `joined`, and `key`. The `showNotification` function, however, isn't reactive because it is defined in the module context of the component. I was only interested in reactively listening to the arguments of the function, not the function itself.

Though it turns out this issue will be fixed in #400 anyway!
